infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -208,7 +211,6 @@ public void start() {
         try {
             log.info("Start Log Replication Discovery Service");
 
-            setLocalNodeId();
```

Comment:
any reason for moving this?

Reply:
yeah, nodeID should come from plugin as it is specific to applications how its computed.